Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -727,7 +727,7 @@
 				CODE_SIGN_ENTITLEMENTS = "Basic-Car-Maintenance/Basic_Car_Maintenance.entitlements";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "\"Basic-Car-Maintenance/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 99CC62WXQ9;
```

Comment:
Can you please resolve these issues, you shouldn't commit your development team or bundle ID
(I did recently fix the project, so it may be easiest to pull `dev` again, and copy / paste your changes over